CMakeLists.txt

```diff
@@ -202,6 +202,12 @@ if (LLAMA_CUBLAS)
     endif()
 endif()
 
+add_library(llama
```

Comment:
This fixes the issue but it's probably better to instead remove the `set_target_properties` block at line 221-223. This way the targets are still defined in the same order.